pynamodb/models.py

```diff
@@ -195,6 +195,10 @@ def __init__(cls, name, bases, attrs):
                         setattr(attr_obj, 'aws_access_key_id', None)
                     if not hasattr(attr_obj, 'aws_secret_access_key'):
                         setattr(attr_obj, 'aws_secret_access_key', None)
+                    if not hasattr(attr_obj, 'ttl_field'):
+                        setattr(attr_obj, 'ttl_field', get_settings_value('ttl_field'))
```

Comment:
Do we want a global "ttl field" override?

Reply:
Yes I deliberately do this.  I do not want it at the attribute level because a table should only have 1 TTL field (ie: dynamo expects 1 TTL field).  Since a table should have 1 TTL field, it should only have 1 TTL second as well.

Reply:
I understand why it's one per model. I'm referring to `get_settings_value` which means "if it's not set on the model, I'll go to a project-level default".